service/quote-server/tdx_lib/src/tdx_lib.rs

```diff
@@ -0,0 +1,387 @@
+/*
+* Copyright (c) 2023, Intel Corporation. All rights reserved.<BR>
+* SPDX-License-Identifier: Apache-2.0
+*/
+
+use anyhow::*;
+use nix::*;
+use std::convert::TryInto;
+use std::fs::File;
+use std::mem;
+use std::os::unix::io::AsRawFd;
+use std::path::Path;
+use std::ptr;
+use std::result::Result;
+use std::result::Result::Ok;
+
+#[repr(C)]
+pub struct tdx10_report_req {
+    subtype: u8,
```

Comment:
A suggestion is to keep the consistency of variable naming conventions, such as words delimited by an underscore.

Reply:
added comments for fields